src/components/EmojiPicker/EmojiPickerButtonDropdown.tsx

```diff
@@ -1,38 +1,45 @@
-import PropTypes from 'prop-types';
 import React, {useEffect, useRef} from 'react';
 import {View} from 'react-native';
+import type {StyleProp, ViewStyle} from 'react-native';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
 import Text from '@components/Text';
 import Tooltip from '@components/Tooltip/PopoverAnchorTooltip';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import type {WithLocalizeProps} from '@components/withLocalize';
+import withLocalize from '@components/withLocalize';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
 import getButtonState from '@libs/getButtonState';
 import * as EmojiPickerAction from '@userActions/EmojiPickerAction';
 import CONST from '@src/CONST';
+import type { Emoji } from '@assets/emojis/types';
 
-const propTypes = {
-    /** Flag to disable the emoji picker button */
-    isDisabled: PropTypes.bool,
-
-    ...withLocalizePropTypes,
-};
+type EmojiSelected = {
+    emojiCode: string;
+    emojiObject: Emoji;
+}
 
-const defaultProps = {
-    isDisabled: false,
-};
+type EmojiPickerButtonDropdownProps = WithLocalizeProps & {
+    isDisabled?: boolean;
+    value: string;
+    style: StyleProp<ViewStyle>;
+    disabled: boolean;
+    onModalHide: () => void;
+    onInputChange: (value: string) => EmojiSelected;
+    // eslint-disable-next-line react/no-unused-prop-types
+    forwardedRef?: React.ForwardedRef<View>;
+}
 
-function EmojiPickerButtonDropdown(props) {
+function EmojiPickerButtonDropdown({isDisabled = false, ...props}: EmojiPickerButtonDropdownProps) {
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const emojiPopoverAnchor = useRef(null);
     useEffect(() => EmojiPickerAction.resetEmojiPopoverAnchor, []);
 
     const onPress = () => {
         if (EmojiPickerAction.isEmojiPickerVisible()) {
-            EmojiPickerAction.hideEmojiPicker();
+            EmojiPickerAction.hideEmojiPicker(false);
```

Comment:
Let's add a default value and leave it without arguments:
```suggestion
            EmojiPickerAction.hideEmojiPicker();
```